src/Kdyby/Replicator/Container.php

```diff
@@ -501,12 +532,12 @@ public function addComponent(Nette\ComponentModel\IComponent $component, $name,
 	public static function register($methodName = 'addDynamic')
 	{
 		if (self::$registered) {
-			Nette\Forms\Container::extensionMethod(self::$registered, function () {
+			Nette\Utils\ObjectMixin::setExtensionMethod('Nette\Forms\Container', self::$registered, function () {
```

Comment:
For controls too https://github.com/nette/forms/commit/d1c80c4e60d8199175f7047d6c172b5d92071a42